checkBoard.c

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
+
+int checkBoard(char board[],char marker) {      	       
+	int i,res;
+//   0   1   2
+//   3   4   5
+//   6   7   8
+	if (marker=='X'){
+		res=1;
+	}else{
+		res=-1;
+	}
+	for (i=0; i<=8; i++) {
+        if (board[i]==marker && board[i+1]==marker && board[i+2]==marker ) {   
```

Comment:
1. This code would accept a win in case of, for example:
```
__X
XX_
___
```
2. It exceeds the boundary.